### DISCUSSION TITLE
[BUG] Add Default Alarm minutes field allows newline character

### DISCUSSION THREAD

**Discussion by zootboy** [bitfireAT/icsx5#138] [General]
I was trying out the new "add default alarm" feature, and I typed in a number in the minutes field and pressed enter. This inserted a newline character in the textbox and caused an error (Introduce a valid number) to appear.

ICSx version: 2.1 from F-droid
Android version: 11 (LineageOS 18.1)
Device: OnePlus 3T

**Comment by ArnyminerZ** [comment]
That error shows up because technically having a line break is introducing an invalid number. You can still remove the line break and type the number correctly and it will work.

However, I agree that "Enter" should simply close the keyboard, and no line breaks should be allowed.
